### ISSUE TITLE
[Request] Profiles linked to site

### ISSUE THREAD

**Issue by Audionut** [notmarek/SendToClient#2]
Having the ability to have profiles linked to a site would be handy.

Seems currently that user must adjust profile via settings if needing to change profile.

**Comment by notmarek** [comment]
Sounds like a great feature!

**Comment by Audionut** [comment]
I can't get this to work here. 
https://ptpimg.me/kgdd5r.png

If I save the profile at one site, then at another site I select another profile and hit save, this new profile gets saved as default at all other sites.

**Comment by notmarek** [comment]
> I can't get this to work here. https://ptpimg.me/kgdd5r.png
> 
> If I save the profile at one site, then at another site I select another profile and hit save, this new profile gets saved as default at all other sites.

I will have to rework the ui but - in the dropdown click "Link to this site" as if its a normal button it should then change to display the sites hostname (if you click the hostname (again like a button) it will remove it from the list)

**Comment by Audionut** [comment]
> it should then change to display the sites hostname

That doesn't happen here.

**Comment by Audionut** [comment]
The other script I linked in the other issue gives a button for dropdown, like this.

https://ptpimg.me/99a9d9.png

I didn't suggest something like this earlier, since it didn't seem to fit with your intentions. But......it does have the advantage of quickly selecting profiles at each torrent, rather than (insert lazy mode) navigating all the way to the script settings.

That is handy, since some sites I would prefer to have 2 profiles at ease, but not a big drama. I like your ui choice of one simple button.

**Comment by notmarek** [comment]
that's odd, works for me ![](https://i.kawaii.sh/WuHEU~b.gif)

what browser and monkey are you using?

**Comment by notmarek** [comment]
> les at each torrent, rather than (insert lazy mode) navigating all the way to the script

that does sound interesting i might be able to add a toggle

**Comment by notmarek** [comment]
> Thorium with violentmonkey. Works as a dropdown here (without actually doing something), not as a button.

are you clicking the thing that pops up from the dropdown?

**Comment by Audionut** [comment]
Yeah, tried that after your button comment, it just closes the dropdown.

**Comment by notmarek** [comment]
> Yeah, tried that after your button comment, it just closes the dropdown.

Odd i will have to rethink the UI then

**Comment by Audionut** [comment]
Tested in latest firefox/arkenfox with violentmonkey. Worked fine. Doesn't work as a button, need to click the dropdown, then click again, but it works.

Last issue must be something specific to Thorium.

**Comment by notmarek** [comment]
> Tested in latest firefox/arkenfox with violentmonkey. Worked fine. Doesn't work as a button, need to click the dropdown, then click again, but it works.
> 
> 
> 
> Last issue must be something specific to Thorium.

I will be redoing the way it works don't worry, i should have some time today :)
